includes/services/Leaflet/jquery.leaflet.js

```diff
@@ -25,7 +25,9 @@
 		* @param {Object} markerData Contains the fields lat, lon, title, text and icon
 		* @return {L.Marker}
 		*/
-		this.addMarker = function (properties) {
+		this.addMarker = function (properties, dontAddToMap) {
```

Comment:
Right...

* addToMap would be nicer, esp than `!dontAddToMap`
* functions ought to do a single thing (on a given level of abstraction). If they take a boolean flag, you know they fail at this